README.md

```diff
@@ -1,3 +1,10 @@
 # Northcoders News API
 
 For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+
+# To run this project locally: 
+ 1. Clone the repo
+ 2. <npm install>
+ 3. create .env.dev & .env.test files in order to successfully connect to the two databases. THEN use the setup-dbs script to create the databases first
+ 
```

Comment:
nice! It might be nice to add what the databases should be called as well - as you'll be connecting to them regularly :)